runtime/tsan.c

```diff
@@ -406,4 +409,26 @@ DEFINE_TSAN_VOLATILE_READ_WRITE(1, 8);
 DEFINE_TSAN_VOLATILE_READ_WRITE(2, 16);
 DEFINE_TSAN_VOLATILE_READ_WRITE(4, 32);
 DEFINE_TSAN_VOLATILE_READ_WRITE(8, 64);
-DEFINE_TSAN_VOLATILE_READ_WRITE(16, 128);
+
+/* We do not treat accesses to 128-bit values as atomic, since it is dubious
+   that they can be treated as such. */
```

Comment:
Are there any volatile accesses to 128-bit values in the runtime? Where do they come from?

Reply:
No, according to my `grep`ping there aren’t any. The 128-bit versions of these functions are there because, without them, building a C library for OCaml with TSan enabled will fail at linking with an unresolved symbol error if it contains `volatile` accesses to 128-bit values. So I figured it would be better to have 128-bit `volatile`s behave silently like plain 128-bit values.

Reply:
Could you add this explanation to the comment?

Reply:
c4e6e30